app/controllers/events_controller.rb

```diff
@@ -33,22 +30,10 @@ def register
       redirect_to event_path(params[:id])
     else
 
-      flash[:error] = "Something went wrong"
+      flash[:error] = 'Something went wrong'
     end
   end
```

Comment:
## Revamp the UI a bit.
- [ ] While at it, kindly work the UI and make sure the link to event creation only shows when the user is signed in.
- [ ] There is no need to show all users on the page. Please fix this.
- [ ] Make sure that only valid navigation links are present at the top of the page. ie, when a user is signed in, only a sign-out link should be present of the two (sign up and sign in) and not both, as well as the create event link. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/navigation_links_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))